src/renderer/features/fellowship-evidence/index.tsx

```diff
@@ -46,7 +49,35 @@ fellowshipEvidenceFeature.inject(evidenceVotingTaskActionSlot, ({ evidence, endB
 });
 
 fellowshipEvidenceFeature.inject(evidenceActionsSlot, ({ evidence }) => {
```

Comment:
How about to hide implementation inside component? This inject should be integration point, not fully functional component